### PR TITLE
Added a management command to reprocess indicators

### PR DESCRIPTION
**Pull request by goyal1092**
## Description
Added a management command to reprocess indicators
Options:
* `--variables :  pass variable ids to process`
* `--datasets : pass dataset ids to process linked variables`
* `--profiles : pass profile ids to process linked variable`
* `--user_id : pass user id to get fail task notification`
* `--rerun_failed : rerun all failed tasks for extraction-ID group which will be created when we run task`
* If no option for variable, dataset & profiles is passed script will run for all existing variables

## Related Issue
https://trello.com/c/qj0bFZj5/778-reprocess-indicators-as-a-sysadmin-i-would-like-to-be-able-to-re-process-indicator-data-so-that-changes-to-the-data-format-can-b

## How to test it locally
run management command : `docker exec -it wazimap-ng_web_1 python3 manage.py reprocess_indicators --variables 20 21 22 28 29 40 45 46`

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out

### Commits

- [x]  commits are clean
- [x]  commit messages are clean

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers
- [x]  black was run locally (as part of the pre-commit hook)

### Testing

- [x]  ✅ added (appropriate) unit tests
- [x]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: wazimap_ng/datasets/management/commands/reprocess_indicators.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        if rerun_failed:
+            to_update = to_update + [t.args[0].id for t in Task.objects.filter(
+                group=f"extraction-{rerun_failed[0]}", success=False
+            )]
+
+        variables = Indicator.objects.all()
+        if to_update:
+            variables = variables.filter(
+                id__in=list(set(to_update))
+            )
```
Comment (jbothma) [inline wazimap_ng/datasets/management/commands/reprocess_indicators.py:73]: Why are there dataset and variable filters?

It's not clear what happens when you don't specify these.

We just want a simple way for sysadmins to safely re-process everything. I think you should remove all the fancy stuff.

All this should do is confirm that they are sure they want to reprocess, and then fire away.

**Comment by jbothma** [comment]
There are no tests.

I think a simple test would be that the sucessful task list is empty, and after calling it, there's one succesful task.

File: wazimap_ng/datasets/management/commands/reprocess_indicators.py (hunk 1 above)
Comment (goyal1092) [reply wazimap_ng/datasets/management/commands/reprocess_indicators.py]: Done 👍

**Comment by goyal1092** [comment]
Added test and updated command @jbothma 👍
